torch/nn/functional.py

```diff
@@ -2950,6 +2950,7 @@ def cross_entropy(
     reduce: Optional[bool] = None,
     reduction: str = "mean",
     label_smoothing: float = 0.0,
+    dim: Optional[int] = None,
```

Comment:
I think we also want this arg for module form `nn.CrossEntropyLoss`, no?

Reply:
@jbschlosser, sure, I'll try adding the arg in ``nn.CrossEntropyLoss`` as well.